SPECS/asciidoc/asciidoc.spec

```diff
@@ -47,11 +55,21 @@ python3 tests/testasciidoc.py run
 %files
 %defattr(-,root,root)
 %license COPYRIGHT
+%doc BUGS.adoc CHANGELOG.adoc README.md
 %{_bindir}/*
-%{_sysconfdir}/*
 %{_mandir}/*
+%{python3_sitelib}/asciidoc/
+%{_pkgdocdir}/doc
+%dir %{python3_sitelib}/asciidoc/resources/filters/latex
+%exclude %{python3_sitelib}/asciidoc/resources/filters/music
+%exclude %{python3_sitelib}/asciidoc-10.2.0-py3.9.egg-info
```

Comment:
Could you brief on these exclusions?

Reply:
I was originally thinking that music filters are not used by anybody as they were not there in the earlier version of rpm and hence excluded them.  
asciidoc-10.2.0-py3.9.egg-info is actually useful for tools like pip and hence keep it. I have added a commit by including both of them now.